tox.ini

```diff
@@ -1,23 +1,23 @@
 # The test environment and commands
 [tox]
 # default environments to run without `-e`
-envlist = py{39,310,311}-{flake8_5,flake8_6}
+envlist = py{39,310,311,312}-{flake8_5,flake8_6}
 
 # create a default testenv, whose behaviour will depend on the name it's called with.
 # for CI you can call with `-e flake8_5,flake8_6` and let the CI handle python version
 [testenv]
 description = Runs pytest, optionally with posargs
 deps =
-    flake8_6: flake8>=6.0
-    flake8_5: flake8>=5.0,<6.0
+    flake8_7: flake8>=7.0
+    flake8_6: flake8>=6.0, <7.0
```

Comment:
Unclear from this whether or not we have flake8_5 and _7 environments - the deps list has changed, but not the envlist (or comment).  

Once you confirm that this is working as intended, including in CI, I'll merge 🙂